pkgs/os-specific/linux/r8125/default.nix

```
@@ -0,0 +1,38 @@
+{ stdenv, lib, fetchFromGitHub, kernel }:
+
+stdenv.mkDerivation rec {
+  name = "r8125-${kernel.version}-${version}";
```

```suggestion
  pname = "r8125";
```

we should probably follow what's already established:
https://repology.org/project/r8125/versions

---

thoughts @timokau @r-ryantm @volth ?

---

@jonringer I changed the name as you suggested.

---

The alternative is that maybe you lag behind the AUR and we could free ride on their updates with nixpkgs-update. The coolest would probably be you work together with their mirror.

---

Oh I see what you mean. I guess it would take the responsibility off my hands.

---

> thoughts @timokau @r-ryantm @volth ?

Yeah I agree. For r8168 I copied the more complex version scheme from somewhere and now repology keeps notifying me about version mismatches. I'll probably remove r8168 soon anyway though, since it would need patching to work with the latest kernel and mainline seems to work good enough now.